src/ign_TEST.cc

```diff
@@ -52,10 +54,43 @@ std::string custom_exec_str(std::string _cmd)
   return result;
 }
 
+using namespace ignition;
+
+class CmdLine : public ::testing::Test
+{
+  // Documentation inherited
+  protected: void SetUp() override
+  {
+    // Change environment variable so that test files aren't written to $HOME
+    common::env(IGN_HOMEDIR, this->realHome);
+    EXPECT_TRUE(common::setenv(IGN_HOMEDIR, this->kFakeHome.c_str()));
+  }
+
+  // Documentation inherited
+  protected: void TearDown() override
+  {
+    // Restore $HOME
+    EXPECT_TRUE(common::setenv(IGN_HOMEDIR, this->realHome.c_str()));
+  }
+
+  /// \brief Directory to act as $HOME for tests
+  public: const std::string kFakeHome = common::joinPaths(PROJECT_BINARY_PATH,
+      "test", "fake_home");
+
+  /// \brief Store user's real $HOME to set it back at the end of tests.
+  public: std::string realHome;
+};
+
 // See https://github.com/ignitionrobotics/ign-gui/issues/75
-TEST(CmdLine, IGN_UTILS_TEST_ENABLED_ONLY_ON_LINUX(list))
+TEST_F(CmdLine, IGN_UTILS_TEST_ENABLED_ONLY_ON_LINUX(list))
 {
+  EXPECT_TRUE(common::removeAll(this->kFakeHome));
```

Comment:
The test is failing here, I'm assuming it's returning false because there is no directory to be removed. I see that we are setting the env variable to `kFakeHome` but I don't see where the directory is created. Is this needed or could we just call `common::removeAll` without the expectation?

Reply:
Ouch, fixed in 6737a83f24916e64772d3e2f75af9b86c2d905fa